xml/System.Reflection.Metadata.Ecma335/MethodSignatureEncoder.xml

```diff
@@ -124,10 +124,10 @@
         <Parameter Name="parameters" Type="System.Action&lt;System.Reflection.Metadata.Ecma335.ParametersEncoder&gt;" Index="2" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
```

Comment:
There should be a type description (like "Provides an encoder for method signatures.")

Reply:
> _There should be a type description (like "Provides an encoder for method signatures.")_

I will add your suggestion, @rpetrusha.
@tmat @nguerrera if you prefer a different summary for the `MethodSignatureEncoder` type, please let me know.